canonical-websites/docs.jujucharms.com
# Pull data from Juju discourse

Instead of the snapcraft one.
Also, switch over to sync workers for Talisker, and tidy up some requirements.

Fixes https://github.com/ubuntudesign/base-squad/issues/430

QA
--

`./run` - see data from discourse.jujucharms.com (even though it's still styled like snapcraft)
